[CI] Version replacement fails in yaml=reference/ml/df-analytics/apis/put-dfanalytics

After version bump to 7.7 a ml docs example fails because the version replacement for the code snippet
doesn't exatly match:

https://gradle-enterprise.elastic.co/s/cefgk52nrjags

Currently seen only on 7.x but issue might also exist on 8.0, only not triggered there because there was no version change there.

```
org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT > test {yaml=reference/ml/df-analytics/apis/put-dfanalytics/line_361} FAILED
    java.lang.AssertionError: Failure at [reference/ml/df-analytics/apis/put-dfanalytics:111]: $body didn't match expected value:
                             $body: 
                    allow_lazy_start: same [false]
                            analysis: 
                     outlier_detection: 
               compute_feature_influence: same [true]
                        outlier_fraction: same [0.05]
                 standardization_enabled: same [true]
                         create_time: same [1579102940510]
                         description: same [Outlier detection on log data]
                                dest: 
                                 index: same [logdata_out]
                         results_field: same [ml]
                                  id: same [loganalytics]
                  model_memory_limit: same [1gb]
                              source: 
                                 index: 
                                       0: same [logdata]
                                 query: 
                               match_all: same [empty map]
                             version: expected String [7.6.0] but was String [7.7.0]

        Caused by:
        java.lang.AssertionError: $body didn't match expected value:
                                 $body: 
                        allow_lazy_start: same [false]
                                analysis: 
                         outlier_detection: 
                   compute_feature_influence: same [true]
                            outlier_fraction: same [0.05]
                     standardization_enabled: same [true]
                             create_time: same [1579102940510]
                             description: same [Outlier detection on log data]
                                    dest: 
                                     index: same [logdata_out]
                             results_field: same [ml]
                                      id: same [loganalytics]
                      model_memory_limit: same [1gb]
                                  source: 
                                     index: 
                                           0: same [logdata]
                                     query: 
                                   match_all: same [empty map]
                                 version: expected String [7.6.0] but was String [7.7.0]
REPRODUCE WITH: ./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/ml/df-analytics/apis/put-dfanalytics/line_361}" -Dtests.seed=DEDD9578431F0ED9 -Dtests.security.manager=true -Dtests.locale=nn-NO -Dtests.timezone=Antarctica/Rothera -Dcompiler.java=13 -Druntime.java=8
```

--- a/docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc
+++ b/docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc
@@ -411,7 +411,7 @@ The API returns the following result:
 }
 ----
 // TESTRESPONSE[s/1562265491319/$body.$_path/]
-// TESTRESPONSE[s/"version": "7.6.0"/"version": $body.version/]
+// TESTRESPONSE[s/"version" : "7.6.0"/"version" : $body.version/]
 
 
 [[ml-put-dfanalytics-example-r]]

